Lib/test/test_builtin.py

```diff
@@ -1226,7 +1364,7 @@ def test_open_default_encoding(self):
             os.environ.clear()
             os.environ.update(old_environ)
 
-    @unittest.skipIf(sys.platform == 'win32', 'TODO: RUSTPYTHON Windows')
+    @support.requires_subprocess()
```

Comment:
need to add back the `skipIf` here apparently, this test continues to fail on windows.